### ISSUE TITLE
Update 3.5.6+2 should be breaking change as minSdkVersion changed

### ISSUE THREAD

**Issue by PcolBP** [darwin-morocho/flutter-facebook-auth#191]
**Describe the bug**
After upgrading plugin to 3.5.6+2 version, error occur: 
```console
uses-sdk:minSdkVersion 20 cannot be smaller than version 21 declared in library [:flutter_facebook_auth] D:\design_book\build\flutter_facebook_auth\intermediates\merged_manifest\release\AndroidManifest.xml as the library might be using APIs not available in 20
        Suggestion: use a compatible library with a minSdk of at most 20,
                or increase this project's minSdk version to at least 21,
                or use tools:overrideLibrary="app.meedu.flutter_facebook_auth" to force usage (may lead to runtime failures)
```

Excuse me but I won't include other information because its useless in this case.

Does Facebook Android SDK 12.1.0 needs API 21 and above?

**Comment by darwin-morocho** [comment]
it doesn't need it but is not recommended support android under 5.0 for that reason I have defined a minimum sdk

**Comment by PcolBP** [comment]
You are right, but still I think that it should be released as breaking change as version 4.0.0

**Comment by darwin-morocho** [comment]
in that case use 3.5.6+3
